docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -208,23 +221,47 @@ version   4.17.0        True        False         5m39s   Cluster version is 4.1
 
 ## Ingress and DNS
 
+As mentioned in the Limitations section, the Ingress and DNS configuration is a manual step that needs to be done after the cluster is created.
+This will be automated in the future.
+
 Once the workload cluster is deploying, the Ingress controller will be installed
 and a router named `router-default` will be created in the `openshift-ingress` namespace.
 
 You'll need to update your DNS with the external IP of that router so Ingress (and dependent operators like console) can work.
-You can run this command to get the external IP:
+
+Once the HostedCluster is created, you need to wait for the `router-default` service to get an external IP:
 
 ```shell
-oc --kubeconfig $CLUSTER_NAME-kubeconfig -n openshift-ingress get service/router-default -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+oc -w --kubeconfig $CLUSTER_NAME-kubeconfig -n openshift-ingress get service/router-default -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
 ```
 
-Now you need to create a DNS A record for `*.apps.<cluster-name>.<base-domain>` that matches the returned IP address.
+When the external IP exists, you can now create a DNS A record for `*.apps.<cluster-name>.<base-domain>` that matches the returned IP address.
+Once this is done, the Ingress operator will become healthy and the console will be accessible shortly after.
+
+!!! note
+
+    The DNS propagation time can vary so you might need to wait a few minutes before your HostedCluster becomes healthy.
+
+At this point, you should be able to access the OpenShift console by navigating to `https://console-openshift-console.apps.<cluster-name>.<base-domain>` in your browser.
+
+To get the `kubeadmin` password, you can run this command:
+```shell
+oc get --namespace clusters Secret/example-kubeadmin-password -o jsonpath='{.data.password}' | base64 --decode
```

Comment:
It wasn't immediately obvious to me that `example` in the above command referred to the cluster name. 

```suggestion
oc get --namespace clusters Secret/${CLUSTER_NAME}-kubeadmin-password -o jsonpath='{.data.password|base64decode}'
```

This also uses the base